content/java/concepts/output/terms/print/print.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: '.print()'
+Description: 'Prints its argument to the console.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Output'
+  - 'Strings'
+  - 'Print'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.print()` method prints its argument to the console. Unlike the similar method [`.println()`](https://www.codecademy.com/resources/docs/java/output/println), `.print()` does not follow its argument with a new line, and any subsequent characters sent to the console will begin wherever the prior `.print()` command left off.
+
+## Syntax
+
+```pseudo
+System.out.print(argument)
+```
+
+`argument` will be displayed on the console.
+
+## Example
+
+The following example prints some content to the console.
+
+```java
+import java.io.*;
+
+public class printExample {
```

Comment:
```suggestion
Public class printExample {
```

Per [this source](https://www.oracle.com/technetwork/java/codeconventions-150003.pdf) on Java class naming (pp. 20 on the PDF)